web/themes/custom/usagov/sass/_uswds-theme-custom-styles.scss

```diff
@@ -946,4 +946,78 @@ div #sm-share a {
   left: 1rem;
   right: 1rem;
   bottom: 0;
+}
+
+// scams reskin
+
+.block-wizardenglish, #block-wizardespanol {
```

Comment:
This looks like it might be a mistake -- do we really have a block-wizardenglish class but use a block-wizardespanol ID?